Automattic/newspack-plugin
# Extract ActionCardSections component

### All Submissions:






### Changes proposed in this Pull Request:



Extracts a `ActionCardSections` component from `PopupGroup`. This way it will be available in `newspack-component` (to be used in `newspack-newsletters`).

Also, a couple of small bugs are fixed:

1. `newspack-card__is-secondary` class - it was not applied to `ActionCard` unless it was a child of `.newspack-card:not( .newspack-card__no-background )`.
2. `Button` with `icon-only` class was missing `display: inline-flex`. It inherits it from `.components-button` in wizards.

### How to test the changes in this Pull Request:

1. On master, visit Campaigns wizard
2. Switch to this branch and rebuild
3. There should be no visual nor functional differences in the campaigns list view

### Other information:
